src/fare-contracts/details/Barcode.tsx

```diff
@@ -240,4 +240,7 @@ const useStyles = StyleSheet.createThemeHook((theme) => ({
     padding: theme.spacings.large,
     backgroundColor: '#FFFFFF',
   },
+  staticQrCode: {
+    aspectRatio: 1.25,
```

Comment:
What's the benefit of increasing the aspect ratio vs reducing the width/height?

Reply:
Might be the wrong usage of aspect ratio here, but it centers things nicely. Might be better to use this?
```css
width: '75%',
marginLeft: 'auto',
marginRight: 'auto',
```

Reply:
I see. The QR code will remain square because of the sizing set on L230. The aspect-ratio increase increases the height of the container it is in, thus making it smaller in width. Is the height affected in terms of scrolling?

Reducing the width and centering should also work. Margin can be reduced to one line if wanted I guess: `margin: '0 auto'`.

Reply:
🤪